core/templates/pages/topic-editor-page/navbar/topic-editor-navbar.component.ts

```diff
@@ -125,7 +128,28 @@ angular.module('oppia').component('topicEditorNavbar', {
         return UndoRedoService.getChangeCount();
       };
 
+      $scope.updateTopicEditorBrowserTabsUnsavedChangesStatus = function() {
```

Comment:
Where is this method used?

Reply:
I have removed this method now. I forgot to remove it earlier.